tools.go

```diff
@@ -0,0 +1,3 @@
+package dns
+
+import _ "golang.org/x/tools/go/packages"
```

Comment:
little comment here on why we have this 3 line file?

Reply:
Will do.